en/appbuilder/generate-app/run-application-locally.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Indigo Design App Builder - Upload Application to GitHub
+_description: Indigo Design App Builder enables users to choose between downloading their application locally or uploading it to their GitHub repository.
+_keywords: App Builder, Web App Builder, Design Systems, Design Systems UX, UI kit, Sketch, Ignite UI for Angular, Sketch to Angular, Angular, Angular Design System, Export code from Sketch, Design Kits for Angular, Sketch UI kits, GitHub
+---
+# Run Application Locally
+
+> [!NOTE]
+><b>The application files can be downloaded as a package and then run in your favourite IDE.
+
+
+### In this article:
+* <a href="#Run-the-app-locally">Run the app locally</a>
+
+### Run the app locally
+To finally develop the app, you don't need to copy paste code. Instead use the generate app button in the toolbar. This quickly creates a package with your Angular application that can be downloaded. To run this app, open up the folder in an IDE like visual studio code. 
+
+* Make sure you have Node.js installed. <br>
+* Then in the console, type `npm install` to install the dependencies. It takes a while to install this, but we don't control this part. <br>
+* Once that is done, type `npm start` to compile and launch the app.<br>
+ 
+ 
+<img class="responsive-img" src="../../images/App-VSCode-Indigo-Design-App-Builder.png" srcset="../../images/App-VSCode-Indigo-Design-App-Builder @2x.png 2x" />
+<p style="text-align:center;">An application running in VSCode</p>
+
+With the ability to create your designs visually using professional grade components in the editor, and then being able to generate an Angular app you can freely extend in your IDE offers you the best of both worlds. In the end, you are guaranteed that what you designed is exactly how it looks when the app is built. Don't forget to share your feedback using the feedback button. We look forward to hearing about your experience and also about what features are valuable to you as we move forwards to our next release milestone in the coming months. 
```

Comment:
I don't have a clue what the first sentence is saying :)

Reply:
```suggestion
The App Builder allows you to design your applications visually using professional grade Ignite UI components, and then you are able to generate the as Angular applications that you can continue developing in your IDE of choice. In the end, you are guaranteed that what you designed in the App Builder design surface looks exactly the same when the app is generated and built. Don't forget to share your feedback using the feedback button. We look forward to hearing about your experience and also about what features are valuable to you as we move forwards to our next release milestones. 
```